.ci/deps.sh

```diff
@@ -6,8 +6,8 @@ deps_linux_alpine() {
   apk upgrade
 
   apk add \
-    git binutils make autoconf automake gcc linux-headers diffutils texinfo \
-    procps socat shadow sudo libgcrypt-dev \
+    git binutils make autoconf automake gcc linux-headers diffutils \
+    procps socat shadow sudo libgcrypt-dev texinfo texlive gzip \
```

Comment:
Pulling in texlive seems overkill, why is this necessary?

Reply:
It won't build the documentation without the `tex` binary ([alpine](https://github.com/hg/tinc/runs/5521836925?check_suite_focus=true#step:8:1190), [debian](https://github.com/hg/tinc/runs/5521837014?check_suite_focus=true#step:8:1081)), and I don't see a more lightweight way of installing it. Whatever package I choose, it pulls around 600 MBs of dependencies.